modules/exoplanets/exoplanetsmodule_lua.inl

```diff
@@ -351,8 +349,9 @@ void createExoplanetSystem(std::string_view starName) {
         "}";
 
         openspace::global::scriptEngine->queueScript(
-            "openspace.addSceneGraphNode(" + planetTrailNode + ");",
-            openspace::scripting::ScriptEngine::RemoteScripting::Yes
+            "openspace.addSceneGraphNode(" + planetTrailNode + ");"
```

Comment:
Could use fmt::format here for a bit more compact way of writing this

Reply:
Please elaborate on how you mean here :) I've been considering using fmt::format for these `queueScript` calls but personally thought this was the more readable option

Also, I prefer these things to be done somewhat consistent at least throughout the file. So maybe let me know how you prefer it to look like and I can change it for all places in the file later.